spec.bs

```diff
@@ -451,8 +451,10 @@ controls which [=origins=] are valid [=aggregation coordinators=]. Every
 <dfn>Default aggregation coordinator</dfn> is an [=aggregation coordinator=]
 that controls which is used for a report if none is explicitly selected.
 
-<dfn>Maximum report contributions</dfn> is a positive integer that controls how
-many contributions can be present in a single report.
+<dfn>Maximum report contributions</dfn> is a [=map=] from [=context type=] to
```

Comment:
I'm a little concerned about the layering here -- in theory, this spec is meant to be as agnostic as possible about the specs using PAA. So, ideally those specs would be the ones defining this constant (and we'd monkeypatch for now), but I'm not sure if there's a good way to do that.

Reply:
Yeah, how do we express this dependency? Basically, we want each calling API to parameterize this spec with a value for *maximum report contributions*?

Reply:
Thinking aloud -- we could maybe change the context type section to say that it's a _struct_ representing the kind of global scope, it contains a name (string) and a default maximum number of contributions (positive integer). (With some comments about the constraints on those values.)